pkgs/development/compilers/flutter/packages/libdeflate/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "libdeflate";
```

Comment:
libdeflate is already packaged in pkg/development/libraries/libdeflate/default.nix with the same version, please use or patch that if you can (it doesn't have outputs specified so you could patch that in)

Reply:
The one in nixpkgs is very old and uses a different build system. It should probably be updated, but I figured that was beyond the scope of this PR.

Reply:
Ah my bad, it has been updated. I'll use it.